### DISCUSSION TITLE
Is it possible to manually choose a cover for every album?

### DISCUSSION THREAD

**Discussion by FuzFuz77** [deckerst/aves#864] [Ideas]
Hello.
Is it possible to manually assign a cover at each picture album, instead of them having the most recent one displayed?

If I may add another suggestion/request, it would be nice if we could export the automatic photos .gifs as gifs, like the google picture app does (I deactivated it and it's quite annoying to reactivate it just to export a gif version of a still picture).

Thank you.

**Comment by deckerst** [comment]
For the album cover it's already possible:
1) go to the Albums page
2) select an album (long press on it)
3) tap on the "Set cover" button at the top
4) select "Custom" in the "Cover" tab

I don't understand what you want to export as a GIF. A static image? A video? An animation?

**Reply by FuzFuz77** [reply]
Thank you for your answer.

I'll elaborate the second question: I have a Pixel 6a phone, when I take a picture with it sometimes the camera automatically creates a small video of the shot (1-2 seconds long), but it's only on the phone and trying to share it results in the phone just sending the still. The Google Pictures app has a function to export that small video as a .gif or .mp4.

**Reply by deckerst** [reply]
Ok, so it's a "motion photo". Aves allows saving or sharing the complete motion photo (image + video), only the still image, or only the video.

For sharing, you can long press on the `Share` quick action in the viewer, and select `Share video only`.
![Screenshot_20240105_144050_Aves~2](https://github.com/deckerst/aves/assets/13898333/4ef72577-643d-4292-ba11-b8c4304ea5cc)

You can also open the video part of the motion photo, from the `Info` page `...` menu > `Open video`. And then do whatever you want with it (share, export...)
![Screenshot_20240105_144255_Aves](https://github.com/deckerst/aves/assets/13898333/d34301ef-edb6-4b98-97a0-cbd95df359a2)

**Comment by FuzFuz77** [comment]
Thank you for your help.
